src/app/pages/admin/admin.controller.js

```diff
@@ -29,8 +30,8 @@ export default class AdminController {
     }
     
     changeUserRole(id){
-        let userRole = this.rootScope.listOfUsers[id].role;
-        userRole = userRole === 'admin' ? 'user': 'admin';
+        // let userRole = this.rootScope.listOfUsers[id].role;
+        const userRole = this.rootScope.listOfUsers[id].role === 'admin' ? 'user': 'admin';
```

Comment:
wat admin.controller do in feature/table directive ?

Reply:
because this small optimization doesn't require new branch

Reply:
remove this change, move to new branch and do refactor, you should set in arguments what is new role!
changeUserRole(id, newrole) {
this.syncDataService.changeUserRole(id, newrole);
}

Reply:
Ok. I've removed this change and will do it on a new branch.